qcengine/programs/tests/test_standard_suite_hf.py

```diff
@@ -158,7 +158,8 @@ def test_sp_hf_rohf(program, basis, keywords, nh2):
 
 
 @pytest.mark.parametrize(
-    "program,basis,keywords", [pytest.param("madness", None, {"dft__k":7,"dft__aobasis": "sto-3g", "dft__econv": 1.0000e-05}),],
+    "program,basis,keywords",
+    [pytest.param("madness", None, {"dft__k": 7, "dft__aobasis": "sto-3g", "dft__econv": 1.0000e-05}),],
```

Comment:
move this to a test_madness.py file since this file is scheduled for termination